spec/models/video_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe Video do
+  it {should belong_to(:category)} 
+  it {should validate_presence_of(:title)}
+  it {should validate_presence_of(:description)}
+
+  describe "search_by_title" do
+    before :each do
+      the_league = Video.create(title: "The League", description: "Bros", small_cover_url: "Videos/TheLeague_small.jpg")
+      futurama = Video.create(title: "Futurama", description: "Cool Story", small_cover_url: "/tmp/futurama.jpg")
+      south_park = Video.create(title: "South Park", description: "Too funny", small_cover_url: "/tmp/south_park.jpg")
+      the_future = Video.create(title: "The Future", description: "Cool Baby", small_cover_url: "/tmp/futurama.jpg")
+
+    end
+
+    it "returns an empty array where no match" do
+      result = Video.search_by_title('boss')
+      expect(result).to eq([])
+    end
+
+    it "returns an array with a single item if there is perfect match" do
+      result = Video.search_by_title('South Park')    
+      expect(result).to eq([South_Park])
```

Comment:
Do your specs pass? You are expecting capitalized (constant) variables here and in spec/models/category_spec.rb. They shouldn't pass.